packages/expression-parser/src/expression-parser/ExpressionParser.js

```diff
@@ -62,6 +83,14 @@ export class ExpressionParser {
     this.validExpressionCache = new Set();
   }
 
+  isCalculation = expression =>
+    expression.length > 0 && expression.startsWith(this.CALCULATION_PREFIX);
+
+  readExpression = expression =>
+    this.CALCULATION_PREFIX
+      ? expression.replace(new RegExp(`^${this.CALCULATION_PREFIX}`), '')
+      : expression;
```

Comment:
Hmmm while I'm not opposed to us doing this, the whole `=` at the start of expressions is really just a report-server thing for now... Indicators and Meditrak App Arithmetic questions both use expression parser, and don't append the `=` to the start to indicate it's a calculation.

To me it feels appropriate to keep all logic about this encapsulated in TransformParser until we've standardised the behaviour across all users of the ExpressionParser?

Reply:
I see what you mean... I just tried to avoid having to override each method in `TransformParser`, to make them aware of the `=` symbol. Not from laziness, but trying to figure out a correct abstraction for this concept...

It seems to me that if a child class does alter how expressions (or a subset of them, eg `calculations`) are defined, then the parent class should expect and allow that... If not, then we are "violating" the contract of the parent class, and is an indication that we shouldn't do it in the first place.

For this specific case: an expression parser does depend on symbols to do its job. Thus, it seems fair to me that a specific implementation of the parser, either via inheritance or via constructor configuration, could use its symbols of choice.